src/app/data-planes/components/PolicyTypeEntryList.vue

```diff
@@ -20,56 +20,71 @@
           <KTable
             class="policy-type-table"
             :fetcher="() => ({ data: policyTypeEntry.connections, total: policyTypeEntry.connections.length })"
-            :headers="tableHeaders"
+            :headers="[
+              { label: 'From', key: 'sourceTags' },
+              { label: 'To', key: 'destinationTags' },
+              { label: 'On', key: 'name' },
+              { label: 'Conf', key: 'config' },
+              { label: 'Origin policies', key: 'origins' },
+            ]"
             :cell-attrs="getCellAttributes"
             disable-pagination
             is-clickable
           >
-            <template #sourceTags="{ rowValue }">
+            <template #sourceTags="{ row }: { row: PolicyTypeEntryConnection }">
```

Comment:
I know I've asked before about using AppCollection here I think you said its because we don't need anything that it gives us here. But just incase, if we were to use it does it render anything that we absolutely don't want, or has an interface that we can'f fill out for reasons? i.e. if we were to use AppCollection would anything bad happen?

Reply:
It would mostly work, but two things that would be wrong:

- there would be different styles for the first column
- if there is any link in the table (which is the case here), that will be triggered by clicking the row

Reply:
Gotcha, thanks!